Gopkg.toml

```diff
@@ -58,7 +58,7 @@
   name = "github.com/getlantern/errors"
 
 [[constraint]]
-  branch = "master"
+  branch = "update-redis-client"
```

Comment:
Is this intended in the PR or an artifact of local testing?

Reply:
intended